Python/Text_Extract/text_extract.py

```diff
@@ -0,0 +1,43 @@
+from PIL import Image
+import pytesseract as pt
+import os
+from pathlib import Path
+
+
+current_location = (os.getcwd() + '\\')
+
+
+def main():
+    # Enter the name of folder which contains img files
+    image_location = input("Enter the Folder name containing Images: ")
```

Comment:
Is it possible to make use of Argument Parsers?

Reply:
It is definitely possible, but I would like to make a case here, Arg parse seems useful when there are multiple tacks where a script can go, here the script is meant for one thing only, I can use Arg Parse but I don't see the reason tbh.

Reply:
Seems reasonable! Thanks for clarifying.